### ISSUE TITLE
Unable to access exported variables from imported http file

### ISSUE THREAD

**Issue by hariprasadiit** [AnWeber/httpyac#81]
I'm trying to access variables exported from the imported file, without success.

Sample

singup.http
```http
POST /signup HTTP/1.1
...headers

{
  "var1": "var_value"
}

{{
exports.varFromSignup = "testValue"
}}
```

init.http
```http
# @import ./signup.http

###
{{
  console.info(varFromSignup) // This line throws error
}}
GET /users/me HTTP/1.1
...headers
```

`console` statement throws error.

Is this expected behaviour? If so, is there a way I can access exported variables in another file?

Thank you

**Comment by AnWeber** [comment]
Only the global variables are imported. In this case `varFromSignup` is not global, because this variable is assigned to the request /signup.
![image](https://user-images.githubusercontent.com/13434739/136784456-65fe30b3-466a-4648-9d13-6510e3a8261f.png)

You need to use `@Ref` or change variable to a global variable
![image](https://user-images.githubusercontent.com/13434739/136784735-80db5a09-eeaf-4f37-9dea-7e39785a45f6.png)

**Comment by AnWeber** [comment]
Would be useful if you can provide an example with httpbin. Will try it on my own.

**Comment by AnWeber** [comment]
In connection with @import/ @ref I have an error, that if the @ref is not present, it comes to an endless loop. I wanted to fix this with httpyac@4.2.0, but it was insufficient. This leads to a crash because of stackoverflow. Maybe this was your problem.
